### PR TITLE
updated dependencies

### PR DESCRIPTION
**Pull request by DarjaLeonova**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,26 +1,26 @@
 module go.expect.digital/translate
 
-go 1.20
+go 1.19
```
Comment (klavinsjanis) [inline go.mod:3]: downgraded go version?

File: Earthfile [hunk 2]
```diff
@@ -3,10 +3,10 @@ VERSION 0.7
 ARG --global USERARCH
 
 ARG --global go_version=1.20.4
```
Comment (VladislavsPerkanuks) [inline Earthfile:5]: 1.20.5

**Comment by VladislavsPerkanuks** [comment]
resolve, merge conflicts, please :)

File: go.mod [hunk 3]
```diff
@@ -3,24 +3,25 @@ module go.expect.digital/translate
...
 	github.com/brianvoe/gofakeit/v6 v6.21.0
 	github.com/go-sql-driver/mysql v1.7.1
 	github.com/google/uuid v1.3.0
-	github.com/grpc-ecosystem/grpc-gateway/v2 v2.15.2
+	github.com/grpc-ecosystem/grpc-gateway/v2 v2.16.0
 	github.com/mitchellh/mapstructure v1.5.0
 	github.com/spf13/cobra v1.7.0
-	github.com/spf13/viper v1.15.0
+	github.com/spf13/viper v1.16.0
 	github.com/stretchr/testify v1.8.3
```
[the remark this answers is not in the document]
Comment (DarjaLeonova) [reply go.mod:14]: Thanks. Strange that 'go get -u ./...' command couldn't update it.

**Comment by DarjaLeonova** [comment]
For some reason, my latest commits to my branch are not showing up here. Looks like some sync issue. 
https://github.com/translate-agent/translate/compare/main...%2375-update-dependencies
If tomorrow morning it will not be visible,I will close this PR and will rise new one.

**Pull request merged**